lib/app/admin/router.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var rebuildData = require('./data-reset');
```

Comment:
If the file is called `data-reset` then the function should be called `resetData` for consistency.